infrastructure/batch/launch_templates.tf

```diff
@@ -11,13 +11,15 @@ resource "aws_launch_template" "data_refinery_lt_standard" {
       delete_on_termination = true
     }
   }
+
+  user_data = base64encode(var.data_refinery_worker_user_data)
 }
 
 resource "aws_launch_template" "data_refinery_lt_bigdisk" {
-  name = "data-refinery-launchtemplate-bigdisk"
+  name = "data-refinery-launchtemplate-bigdisk-${var.user}-${var.stage}"
   tags = var.default_tags
   # ccdl-data-refinery-base-v2.0 image
-  image_id = "ami-01c08d4de548477df"
+  image_id = "ami-0f9b369e4ad339b15"
```

Comment:
maybe these should be config variables?

Reply:
mmm that's a good call.

Reply:
I've made this change locally. I'll include it in a later PR. (easier to test that way)